src/data/mv_polynomial/basic.lean

```diff
@@ -557,6 +563,10 @@ add_monoid_algebra.support_mul_single p _ (by simp) _
   (X s * p).support = p.support.map (add_left_embedding (single s 1)) :=
 add_monoid_algebra.support_single_mul p _ (by simp) _
 
+@[simp] lemma support_smul_eq {S₁: Type*} [semiring S₁] [module S₁ R] [no_zero_smul_divisors S₁ R]
```

Comment:
```suggestion
@[simp] lemma support_smul_eq {S₁ : Type*} [semiring S₁] [module S₁ R] [no_zero_smul_divisors S₁ R]
```

Reply:
Oh! Sorry about that. (The habit from python...)